xml/System.Linq/Enumerable.xml

```diff
@@ -820,9 +810,9 @@
 ## Examples  
```

Comment:
For the Enumerable.Average, the Func examples can also be interactive.

Also, lines 745-753 should be removed. This is still showing up:
![image](https://user-images.githubusercontent.com/12971179/107302948-03873100-6a33-11eb-812a-4f266e4529a1.png)

Reply:
For applying interactive in Func examples the following comment will apply?
_"Remove the Examples section entirely for content not containing Func if they have a warning that the code example uses an overload that is different from the specific overload that this topic describes."_